### PR TITLE
llzip

### PR DESCRIPTION
**Pull request by ZenAtomsk**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/llZip/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+## Approach & Efficiency
+
+Make LL2.next equal LL1.next then make LL1.next LL2 current through a loop.
+
+## Solution
+
+- [LL Zip](ll-zip.js) 
+
+![Whiteboard](assets/ll-zip.png)
```
Comment (adrianhuebner) [inline javascript/code-challenges/llZip/README.md:16]: What a great whiteboard!
I would like to see some tests added as well!
